src/components/Card/Card.tsx

```diff
@@ -13,6 +13,12 @@ import {Popover} from '../Popover';
 import {Header, Section, Subsection} from './components';
 import styles from './Card.scss';
 
+export type {
+  SectionProps as CardSectionProps,
```

Comment:
Polaris tends to avoid renaming when reexporting as it having the same concept available with two different names is confusing. Can you rename HeaderProps/SectionProps and SubsectionProps at their source so this line can be just `export type {CardHeaderProps, CardSectionProps, CardSubSectionprops} from ...`

Reply:
Isn't it going to cause confusion when importing them?
```tsx
import type {HeaderProps} from '@shopify/polaris';
``` 
In my view, this ☝️ implies that we have a `Header` component in place and we're importing its Prop types. The same applies to `SectionProps` and `SubSectionprops`. Alternatively, I can rename its actual types.